bower.json

```diff
@@ -10,6 +10,9 @@
     "pretender": "1.4.2",
     "prism": "1.6.0",
     "showdown": "1.5.1",
-    "sinonjs": "1.17.1"
+    "sinonjs": "1.17.1",
+    "moment": "^2.17.1",
+    "pikaday": "~1.5.1",
+    "clockpicker-seconds": "~0.1.6"
```

Comment:
I'm strongly against bringing in more bower dependencies. I've been trying to eliminate them for a while as has Ember itself. IMO these really need to be shimmed so we can include them via NPM instead, I'll even help make that happen.